sentry_sdk/envelope.py

```diff
@@ -230,22 +223,27 @@ def __repr__(self):
     @property
     def data_category(self):
         # type: (...) -> EventDataCategory
-        rv = "default"  # type: Any
-        event = self.get_event()
-        if event is not None:
-            rv = get_event_data_category(event)
+        ty = self.headers.get("type")
+        if ty == "session":
+            return "session"
+        elif ty == "attachment":
+            return "attachment"
+        elif ty == "transaction":
+            return "transaction"
+        elif ty == "event":
+            return "error"
         else:
-            ty = self.headers.get("type")
-            if ty in ("session", "attachment"):
-                rv = ty
-        return rv
+            return "default"
 
     def get_bytes(self):
         # type: (...) -> bytes
         return self.payload.get_bytes()
 
     def get_event(self):
         # type: (...) -> Optional[Event]
+        """
+        Returns a regular (non-transaction) event if there is one.
```

Comment:
"regular non-transaction event" is weird. Does this mean

```suggestion
        Returns an error event if there is one.
```

?